pom.xml

```diff
@@ -48,12 +48,17 @@
   </pluginRepositories>
 
   <dependencies>
-        <dependency>
-            <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>token-macro</artifactId>
-            <version>1.10</version>
-        </dependency>
-    </dependencies>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>docker-commons</artifactId>
+      <version>1.0-alpha-5</version>
```

Comment:
Update it, since there has been an incompatible API change (`KeyMaterialFactory` interjected).